src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -52,7 +52,7 @@
                     setTimeout(function(){
                         options.maxTries--;
                         request(url, onSuccess, onError, payload, options);
-                    }, 1000);
+                    }, Math.round(5 / options.maxTries) * 1000);
```

Comment:
this will break if `maxTries` is bigger than `10` as `5 / maxTries` will be lower than `0.5` and so rounding will give `0`

Reply:
Hi, @stof. Yes, you're right. Would it be okay if I used `Math.ceil` instead? Then the wait sequence would be (with maxTries starting at 5): 1, 2, 2, 3, 5

Reply:
I have used `Math.ceil` instead now :)

Reply:
to me, it looks weird to implement that by taking `5 / remaining_retries`.
It would make more sense to compute that delay based on the number of attempts already done, not based on the number of remaining ones.

Reply:
I totally agree - that is also what every other backoff strategy does it.

I just used this approach to not make the change an overkill (in my eyes). By looking at commits it seems that the `maxTries` number hasn't changed for years and probably won't (?) change for years to come.